src/containers/ContentLoader.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import ContentLoader from "react-content-loader";
+
+export function ChartLoader({title = 'Loading Data', backgroundColor = "#f3f3f3", 
+	foregroundColor = "#ecebeb", speed = 1.2, width = "100%", height = 150 }, props) {
+
+	let rectangleQty = new Array(55).fill(1);
+
+	return (
+			<ContentLoader 
+				title={title}
+				speed={speed}
+				width={width}
+				height={height}
+				viewBox={`0 0 ${width} ${height}`}
+				backgroundColor={backgroundColor}
+				foregroundColor={foregroundColor}
+				{...props}
+			>
+				{
+					rectangleQty.map((el, i) =>  {
+						let rand = Math.floor(Math.random() * (height * 0.7))
+						return (
+							<rect x={20 * i} y={rand} rx="5" ry="5" width="10" height={height-rand} key={i}/> 
+							)
+					})
+				}
+			</ContentLoader>
+	);
+}
+
+export function CellContentLoader({rows = 10, cols = 4, title = 'Loading Data',
+	backgroundColor = "#E4E5E6", foregroundColor = "#ffffff", speed = 1.2, header = true, size = 'lg' }, props) {
+
+	let rectangles = [],
+		width = size === 'lg' ? 900 : 400,
+		headRect = [[0, 0, width, 17],
+					[0, 35, width, 17]],
```

Comment:
As I see you used here too many tabs for those arguments. Also you should break line after first `[` as well:
```js
headRect = [
  [0, 0, width, 17],
  [0, 35, width, 17]
]
```